package.json

```diff
@@ -28,9 +28,9 @@
   },
   "homepage": "https://github.com/PerimeterX/perimeterx-node-express#readme",
   "dependencies": {
-    "perimeterx-node-core": "~1.0.0",
-    "cookie-parser": "1.4.1",
-    "moment": "^2.19.3"
+    "cookie-parser": "^1.4.1",
```

Comment:
since we are now using cookie-parser from our code, it can be removed from the requirements section on the readme file